app/kubemci/pkg/gcp/forwardingrule/forwardingrulesyncer.go

```diff
@@ -165,11 +189,12 @@ func (s *ForwardingRuleSyncer) desiredForwardingRule(lbName, ipAddress, targetPr
 	}
 	// Compute the desired forwarding rule.
 	return &compute.ForwardingRule{
-		Name:        s.namer.HttpForwardingRuleName(),
-		Description: desc,
-		IPAddress:   ipAddress,
-		Target:      targetProxyLink,
-		PortRange:   httpDefaultPortRange,
-		IPProtocol:  "TCP",
+		Name:                s.namer.HttpForwardingRuleName(),
+		Description:         desc,
+		IPAddress:           ipAddress,
+		Target:              targetProxyLink,
+		PortRange:           httpDefaultPortRange,
+		IPProtocol:          "TCP",
+		LoadBalancingScheme: "EXTERNAL",
```

Comment:
Why is this required?
fwiw, we dont set it in ingress-gce code

Reply:
This was one of the mismatches when comparing desired and existing. The existing one had EXTERNAL in it... (would you rather I do some manual checking against LoadBalancingScheme in forwardingRuleMatches?

Reply:
hmm looks like its the default value. Fine as is.